src/components/Buttons/Bubble/Bubble.jsx

```diff
@@ -0,0 +1,12 @@
+import { BubbleStyled } from '@/components/Buttons/Bubble/BubbleStyled';
+
+const Bubble = props => {
+  const { text, ...position } = props;
+  return (
+    <BubbleStyled left={position.left} marginbottom={position.marginbottom}>
```

Comment:
Styled components custom props should start with '$' https://arc.net/l/quote/qzsatkhv

Reply:
thank you, didn't knew that. fixed.